TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/controller/OnDirectionPressedListener.java

```diff
@@ -0,0 +1,28 @@
+package com.novoda.tpbot.controller;
+
+import com.novoda.notils.logger.simple.Log;
+
+interface OnDirectionPressedListener {
```

Comment:
What about a simpler `DirectionListener` or `ControlListener`? 
It is not just handling the "pressed" part, but also the "released" and the lazerzzzzz